internal/httpapi/httpapi.go

```diff
@@ -28,60 +31,73 @@ const (
 	statusAPIPathReady         = "/v1/status/ready"
 	ingestAPIPath              = "/v1/data"
 	ingestAPIPathReady         = "/v1/data/ready"
-	readinessProbeRetryBackoff = 10 * time.Millisecond
+	readinessProbeRetryBackoff = 100 * time.Millisecond
```

Comment:
out of curiosity, why increase it?

Reply:
It was a bit of a sneaky change, sorry about that! The main motivation was that during tests, the probing mechanism was consuming quite a bunch of CPU. Since we're waiting for network requests to succeed, the 100ms backoff did not seem that much. I can always revert it back to 10ms if you prefer it :)